lib/benches/grounding_space.rs

```diff
@@ -24,7 +25,7 @@ fn query_x10(bencher: &mut Bencher) {
     let space = space(10);
     bencher.iter(|| {
         let res = space.query(&expr!("=" ("func-9" "arg") X));
-        assert_eq!(res, vec![bind!{ X: Atom::sym("arg") }]);
+        assert_eq!(res, BindingsSet::from(bind!{ X: Atom::sym("arg") }));
```

Comment:
I see this pattern is frequently met in code. May be we need adding another macro `bind_set!` to construct `BindingSet` from `bind!` arguments?

Reply:
Adding bind_set! macro & documentation.  Updated to use where appropriate.